src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -95,6 +98,47 @@ private static void setupLogger(Application application) {
 		application.addComponentMenuElement(Logger.class, "Severe level",
 				(ActionEvent e) -> logger.setLevel(Level.SEVERE));
 		application.addComponentMenuElement(Logger.class, "OFF logging", (ActionEvent e) -> logger.setLevel(Level.OFF));
+
+
+		application.getFreePanel().addMouseListener(new MouseListener(){
```

Comment:
extract to src/main (separate class and configuration routine)

Reply:
resolved